src/test/java/org/eclipse/esmf/ame/services/ModelServiceTest.java

```diff
@@ -187,24 +187,25 @@ void testGetAllNamespacesThrowsIOException() {
       assertThrows( FileNotFoundException.class, () -> modelService.getAllNamespaces( true, validationProcess ) );
    }
 
-   @Test
-   void testMigrateModel() throws IOException {
-      try ( final MockedStatic<LocalFolderResolverStrategy> utilities = Mockito.mockStatic(
-            LocalFolderResolverStrategy.class ) ) {
-         final Path storagePath = Path.of( openManufacturingTestPath.toString(), oldAspectModelFile );
-
-         utilities.when( () -> LocalFolderResolverStrategy.transformToValidModelDirectory( any() ) )
-                  .thenReturn( storagePath.toString() );
-
-         final ValidationProcess validationProcess = Mockito.mock( ValidationProcess.class );
-         Mockito.when( validationProcess.getPath() ).thenReturn( storagePath );
-
-         final String migratedModel = modelService.migrateModel(
-               Files.readString( storagePath, StandardCharsets.UTF_8 ), validationProcess );
-
-         checkMigratedModel( migratedModel );
-      }
-   }
+   //   TODO should be reactivated as soon as there is something to migrate again.
```

Comment:
Uncomment this test and add `@Disabled` annotation with an explanation on it. This makes sure that if something that is used in the method is refactored/renamed, the test code is updated.